ui/components/src/test/propertygrid/dataproviders/filterers/CategoryPropertyDataFilterer.test.ts

```diff
@@ -0,0 +1,162 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { expect } from "chai";
+import * as faker from "faker";
+import sinon from "sinon";
+import { CategoryPropertyDataFilterer } from "../../../../ui-components/propertygrid/dataproviders/filterers/CategoryPropertyDataFilterer";
+import { PropertyCategory } from "../../../../ui-components/propertygrid/PropertyDataProvider";
+
+describe("CategoryPropertyDataFilterer", () => {
+
+  describe("When filter text not set", () => {
+    const categoryToTest: PropertyCategory =
+    {
+      name: "Cat1", label: "Category 1", expand: true, childCategories: [
+        { name: "Cat1-1", label: "Category 1-1", expand: true, childCategories: [] },
+      ],
+    };
+
+    it(`Should return empty string`, () => {
```

Comment:
Not clear what "should return empty string". Maybe these tests should be moved under `describe("[get] filterText", () => {})`?

Reply:
Created mentioned describe for all the text filterers